srcpkgs/nyxt/template

```diff
@@ -12,14 +11,19 @@ short_desc="Keyboard-oriented, extensible web-browser"
 maintainer="0x0f0f0f <sudo-woodo3@protonmail.com>"
 license="BSD-3-Clause"
 homepage="https://nyxt.atlas.engineer/"
-distfiles="https://nyxt.atlas.engineer/static/release/nyxt-linux-${version}.tar.gz"
-checksum=71154bc6ef88f176f3f7783e1c0ecc191b6dcdd996e1c67f2bbc417b651e0355
+distfiles="https://github.com/atlas-engineer/nyxt/releases/download/${version}/${pkgname}-${version}-source-with-submodules.tar.xz"
+checksum=f611a0cee412b64f4bf93228aa975621734ace797153580c58c61616c7db02d8
 # Disable check because ASDF/USER::PROVE is not installed
 make_check=no
 nostrip=yes
 nopie=yes
 nocross="Cross-compilation fails due to package iolib/syscalls setting incorrect compiler flags"
 
+do_extract() {
+	mkdir -p ${wrksrc}
+	bsdtar xvf ${XBPS_SRCDISTDIR}/${pkgname}-${version}/${pkgname}-${version}-source-with-submodules.tar.xz -C ${wrksrc}
+}
```

Comment:
See https://github.com/void-linux/void-packages/pull/33871/files#r742124959

Reply:
Yes, it is the same issue i solved with do_extract().
Both, 2.2.2 and 2.2.3 need this fix.

Reply:
> Set create_wrksrc instead.

It should be possible to delete do_extract() and replace it with create_wrksrc and wrksrc.

@WilliamBehrens could you take a quick look at this PR?

Reply:
yes, `create_wrksrc` works.